[2024-04-15T16:34:14.158+0000] {taskinstance.py:1957} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dbt_visualization.generate_visualizations scheduled__2024-04-14T00:00:00+00:00 [queued]>
[2024-04-15T16:34:14.164+0000] {taskinstance.py:1957} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dbt_visualization.generate_visualizations scheduled__2024-04-14T00:00:00+00:00 [queued]>
[2024-04-15T16:34:14.164+0000] {taskinstance.py:2171} INFO - Starting attempt 15 of 16
[2024-04-15T16:34:14.179+0000] {taskinstance.py:2192} INFO - Executing <Task(PythonOperator): generate_visualizations> on 2024-04-14 00:00:00+00:00
[2024-04-15T16:34:14.185+0000] {standard_task_runner.py:60} INFO - Started process 3529 to run task
[2024-04-15T16:34:14.188+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'dbt_visualization', 'generate_visualizations', 'scheduled__2024-04-14T00:00:00+00:00', '--job-id', '110', '--raw', '--subdir', 'DAGS_FOLDER/dbt_visualization.py', '--cfg-path', '/tmp/tmpmdwyc33f']
[2024-04-15T16:34:14.190+0000] {standard_task_runner.py:88} INFO - Job 110: Subtask generate_visualizations
[2024-04-15T16:34:14.228+0000] {task_command.py:423} INFO - Running <TaskInstance: dbt_visualization.generate_visualizations scheduled__2024-04-14T00:00:00+00:00 [running]> on host ec3d883e7b78
[2024-04-15T16:34:14.292+0000] {taskinstance.py:2481} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='66130047@dpu.ac.th' AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='dbt_visualization' AIRFLOW_CTX_TASK_ID='generate_visualizations' AIRFLOW_CTX_EXECUTION_DATE='2024-04-14T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='15' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-04-14T00:00:00+00:00'
[2024-04-15T16:34:14.299+0000] {base.py:83} INFO - Using connection ID 'my_***_conn' for task execution.
[2024-04-15T16:34:14.310+0000] {logging_mixin.py:188} WARNING - /opt/***/dags/dbt_visualization.py:27 UserWarning: pandas only supports SQLAlchemy connectable (engine/connection) or database string URI or sqlite3 DBAPI2 connection. Other DBAPI2 objects are not tested. Please consider using SQLAlchemy.
[2024-04-15T16:34:14.321+0000] {logging_mixin.py:188} INFO - my df                                                title  ...   post_date
0                                              Facts  ...  2024-02-04
1                           "We have great datasets"  ...  2023-06-08
2                                  i just want sleep  ...  2023-04-20
3      I've finally built the perfect data pipeline!  ...  2023-09-19
4           PSA: Learn Vendor Agnostic Technologies!  ...  2023-04-26
5  Had an onsite interview with one of FAANG, all...  ...  2024-02-16
6  The data engineer came to me... tears in his eyes  ...  2023-07-26
7   Exporting to excel is always a people pleaser...  ...  2023-04-14
8                                Itâ€™s happening guys  ...  2024-03-12
9                                          2022 Mood  ...  2022-01-09

[10 rows x 4 columns]
[2024-04-15T16:34:14.381+0000] {python.py:201} INFO - Done. Returned value was: None
[2024-04-15T16:34:14.392+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=dbt_visualization, task_id=generate_visualizations, execution_date=20240414T000000, start_date=20240415T163414, end_date=20240415T163414
[2024-04-15T16:34:14.440+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-04-15T16:34:14.454+0000] {taskinstance.py:3281} INFO - 0 downstream tasks scheduled from follow-on schedule check
